docs/apache-airflow/production-deployment.rst

```diff
@@ -118,852 +118,7 @@ To mitigate these issues, make sure you have a :doc:`health check </logging-moni
 Production Container Images
 ===========================
 
-Production-ready reference Image
```

Comment:
Do we have a redirect for this page to the new location?

Reply:
This page still exists, so we can't redirect the user. Instead of old content with multiple sections, we now have one section with an incentive with a link to the new package.